addons/spectator/stringtable.xml

```diff
@@ -103,6 +104,7 @@
         <!-- Interface strings -->
         <Key ID="STR_ACE_Spectator_UnitTitle">
             <English>Spectator Units</English>
+            <Polish>Obserwatorzy</Polish>
```

Comment:
Hmm, perhaps this string needs to eb adjusted. It's actually the heading of the spectat*able* units list in the UI. I suspect your translation here means "observers" based on the spelling